examples/objective-c/helloworld/main.m

```diff
@@ -25,20 +25,41 @@
 
 static NSString * const kHostAddress = @"localhost:50051";
 
+@interface HLWResponseHandler : NSObject<GRPCProtoResponseHandler>
+
+@end
+
+// A response handler that only
+@implementation HLWResponseHandler
+
+- (dispatch_queue_t)dispatchQueue {
+  return dispatch_get_main_queue();
+}
+
+- (void)didReceiveProtoMessage:(GPBMessage *)message {
+  NSLog(@"%@", message);
```

Comment:
What will get logged here? Don't we need to cast message to `HLWReply*` and log `response.message`?

Reply:
This logs the entire proto object, including response.message and all other information in this object.